coindecko/pyscript/ingest_data.py

```diff
@@ -1,49 +1,47 @@
 import requests
 import pandas as pd
 
-## Waiting until thingss break
-per_page=250
+# Waiting until things break
+per_page = 250
 page = 1
 json_df = []
 
-### 
+# Write a function to test if the first id is bitcoin, if it's true the convert the data to a dataframe
+
+
+def test_data_ingested(df):
+    if df[0]['id'] == 'bitcoin':
+        df = pd.DataFrame(df, columns=['id', 'name', 'current_price', 'total_volume', 'market_cap', 'last_updated'])
+        return df
+    else:
+        print('Wrong data imported! Kindly check the data source')
+
+
 while True:
```

Comment:
The test_data_ingested() function doesn't return a boolean value? I think that's what you're trying for here but we can further review this piece.

Reply:
Sure, we can further review it.